2d_array.c

```diff
@@ -2,6 +2,7 @@
 int main()
 {
     //this is test comment from subhash.
+    //this is another comment.
```

Comment:
Please explain

Reply:
this is a simple comment, further explanation is not required